modules/virt-attaching-vm-secondary-network-cli.adoc

```diff
@@ -45,7 +45,7 @@ spec:
 ----
 <1> The name of the bridge interface.
 <2> The name of the network. This value must match the `name` value of the corresponding `spec.template.spec.domain.devices.interfaces` entry.
-<3> The name of the network attachment definition, prefixed by the namespace where it exists. The namespace must be either the `default` namespace or the same namespace where the VM is to be created.
+<3> The name of the network attachment definition, prefixed by the namespace where it exists. The namespace must be either the `default` namespace or the same namespace where the VM is to be created. In this case, `multus` is used. Multus is a "meta" cloud network interface (CNI) plug-in that allows multiple CNIs to exist so that a pod or virtual machine can use the interfaces it needs.
```

Comment:
Is "meta" really necessary or can it be removed? If you keep it, then I'd suggest to remove the quotes at least.

Reply:
I think it's TMI --- the user either knows this or can find it out from Multus.